TrafficCapture/dockerSolution/src/main/docker/docker-compose.yml

```diff
@@ -105,6 +105,8 @@ services:
       - MIGRATION_KAFKA_BROKER_ENDPOINTS=kafka:9092
       - AWS_PROFILE=default
 #    command: ./runTestBenchmarks.sh
+    ports:
```

Comment:
I switched this around, that's what was causing the E2E failure.

I wanted just one place to comment/uncomment in order to enable/disable the api. Any reason we can't keep it enabled by default or should i update the comment to mention both places to enable

Reply:
Since we are considering Django to be an experimental feature right now, I feel better with just having a comment saying that these two places need to be updated

Reply:
Just addressed with disabled by default